packit_service/worker/jobs.py

```diff
@@ -640,7 +640,16 @@ def check_explicit_matching(self) -> List[JobConfig]:
         Returns:
             List of job configs.
         """
-        matching_jobs = []
+
+        def compare_jobs(a, b):
```

Comment:
```suggestion
        def compare_jobs_without_triggers(a, b):
```
to be explicit about it